controllers/socket/socket.go

```diff
@@ -31,12 +33,7 @@ func SocketInit(so socketio.Socket) {
 	})
 
 	log.Println("on connection")
-	so.Join("chat")
-	so.On("chat message", func(msg string) {
-		log.Println("emit:", so.Emit("chat message", msg))
-		so.BroadcastTo("chat", "chat message", msg)
-	})
-
+	so.Join("-1") //room for global chat
```

Comment:
You don't need a room for this, you can just broadcast the global chat messages to everyone